src/common/utils/setup-helpers.js

```diff
@@ -318,6 +319,24 @@ function checkNativeTextTracksSupport(options: KPOptionsObject): void {
   }
 }
 
+/**
+ * Sets config option fullscreen element for Vr Mode support
+ * @private
+ * @param {KPOptionsObject} options - kaltura player options
+ * @returns {void}
+ */
+function configureVrRelevantOptions(options: KPOptionsObject): void {
```

Comment:
I would say `Relevant` is redundant as any config is relevant per-se.

Reply:
I'll change it to default instead

Reply:
I'll change it `to configureVrDefaultOptions ` instead